crypto/conf/conf_mod.c

```diff
@@ -11,6 +11,7 @@
 #include <ctype.h>
 #include <openssl/crypto.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
On *side note* this one seems to be dependency on OPENSSL_CONF which isn't OS-specific.